### ISSUE TITLE
Getting the following Error when Loading the mod on 1.0 B333

### ISSUE THREAD

**Issue by toxicdust99** [Byteblazar/mods#1]
2024-08-14T00:39:29 29.578 WRN XML patch for "entityclasses.xml" from mod "zByteblazars_Screamers" did not apply: <remove xpath="/entity_classes/entity_class[@name='zombieScreamer' or @name='zombieGuppyLucy']/property[@name='AIFeralSense']"  (line 3 at pos 2)
2024-08-14T00:39:29 29.578 WRN XML patch for "entityclasses.xml" from mod "zByteblazars_Screamers" did not apply: <remove xpath="/entity_classes/entity_class[@name='zombieScreamer' or @name='zombieGuppyLucy']/property[@name='AINoiseSeekDist']"  (line 4 at pos 2)
2024-08-14T00:39:29 29.580 WRN XML patch for "entityclasses.xml" from mod "zByteblazars_Screamers" did not apply: <remove xpath="/entity_classes/entity_class[@name='zombieScreamer' or @name='zombieGuppyLucy']/property[@name='LookAtAngle']"  (line 8 at pos 2)
2024-08-14T00:39:29 29.580 WRN XML patch for "entityclasses.xml" from mod "zByteblazars_Screamers" did not apply: <remove xpath="/entity_classes/entity_class[@name='zombieScreamer']/property[@name='AITask']"  (line 10 at pos 2)
2024-08-14T00:39:29 29.581 WRN XML patch for "entityclasses.xml" from mod "zByteblazars_Screamers" did not apply: <remove xpath="/entity_classes/entity_class[@name='zombieScreamer']/property[@name='AITarget']"  (line 11 at pos 2)

**Comment by Byteblazar** [comment]
TL;DR: Safe to ignore. If you want to test the mod, open the console and enter `spawnscouts`, then sit around for a minute until screamers show up at your location.

Those yellow warnings are harmless. They happen for two reasons:

1. My mod tries to patch zombieGuppyLucy if you have Guppy's zombies mod installed, but if you don't (which is fine because it's optional), the game shows it couldn't patch it.
2. The way I wrote the mod, it removes entries before adding my changes, because that's better for compatibility with other mods. Basically, if my mod loads after another mod that also happens to modify some or all of the same parameters, my mod will win the conflict (superseding the changes applied by the other mod), and there won't be any problems when you play, but if you don't have any other mods that mess with screamers, the game will show these yellow messages on the console because it couldn't remove anything before applying my changes (because there was nothing there to remove).

**Comment by Byteblazar** [comment]
Shouldn't be any yellow warnings now that I moved things into patches. Closing this.
